Ollama doc review (#34)
timescale/pgai: docs/ollama.md

# Use pgai with Ollama

This page shows you how to:

- [Configure pgai for Ollama](#configure-pgai-for-ollama)
- [Add AI functionality to your database](#usage)

## Configure pgai for Ollama

> To use pgai with Ollama, Ollama must be running and network-accessible to your Timescale Cloud service or 
> self-hosted database.

Reviewer: ```suggestion
To use pgai with Ollama, Ollama must be running and network-accessible to your database.
```

> To specify the Ollama network address, Ollama functions in pgai use the `_host` parameter. 

Reviewer: ```suggestion
To specify the Ollama network address, Ollama functions in pgai use the `_host` parameter. Alternatively, the `ai.ollama_host` config setting can used.
```

> If you have not set `_host`, the `ai.ollama_host` config setting is missing. pgai defaults to 
> `http://localhost:11434`, and a warning is appended to to the log file.

Reviewer: ```suggestion
If you do not pass a `_host` argument, and the `ai.ollama_host` config setting is missing. pgai defaults to 
`http://localhost:11434`, and a warning is appended to the log file.
```
